wire-runtime/src/main/java/com/squareup/wire/WireOutput.java

```diff
@@ -50,11 +50,12 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 import java.io.IOException;
+import okio.BufferedSink;
 
 /**
  * Utilities for encoding and writing protocol message fields.
  */
-public final class WireOutput {
```

Comment:
Yep. That's what I planned. Was trying to make this a small change but it kind of got out of control so I'll probably go all in. There's a couple small tweaks like this all over for reducing allocations.